app/TuistApp/Sources/Views/PhysicalDeviceRow/PhysicalDeviceRow.swift

```diff
@@ -45,9 +45,15 @@ struct PhysicalDeviceRow: View {
             VStack(alignment: .leading) {
                 Text(device.name)
                     .font(.title3)
-                Text(device.osVersion)
+                device.osVersion.map { Text($0) }
             }
             Spacer()
+
+            if device.transportType == .wifi {
```

Comment:
What do you think of using an exhaustive switch, so that the compiler warns about an unhandled case when we add a new type as we'd most likely want to add an icon for it as well?